holidayextras/brands
# Updating brands to reference newly named npm packages

#### What does this PR do? (please provide any background)
Updates all the packages to used the newly named npm modules. This cannot me merged until Ollie has created the packages.

**Add `circlecihxagent`as maintainer to npm packages before this is merged.

#### What tests does this PR have?
None

#### How can this be tested?
It can't unfortunately. Upon merge to master you can see the build script publishing each individual brand.

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
